resources/js/Pages/Admin/Invite.vue

```diff
@@ -0,0 +1,144 @@
+<script setup lang="ts">
+import { defineProps, ref, computed } from 'vue'
+import FormButton from '@/components/Common/FormButton.vue'
+import { useForm } from '@inertiajs/vue3'
+import { useParams } from '@/Helpers/Params'
+
+const sortOptions = [
+  {
+    name: 'ID',
+    value: 'id',
+  },
+  {
+    name: 'Imię',
+    value: 'name',
+  },
+  {
+    name: 'Szkoła',
+    value: 'school',
+  },
+]
+
+const orderOptions = [
+  { name: 'Rosnąco',
+    value: 'asc',
+  },
+  { name: 'Malejąco',
+    value: 'desc',
+  },
+]
+
+const props = defineProps<{
+  users: Pagination<User>
+  quiz: Quiz
+  schools: {
+    id: number
+    name: string
+    city: string
+  }
+  assigned: number[]
+}>()
+
+console.log(props)
+const params = useParams()
+
+const form = useForm({
+  search: params.name ?? '',
+  sort: params.sort ?? 'id',
+  order: params.order ?? 'asc',
+  schoolId: params.schoolId ?? null,
+  limit: params.limit ?? null,
+})
+
+const selectedUserIds = ref<number[]>([])
+
+const toggleUserSelection = (userId: number) => {
+  if (selectedUserIds.value.includes(userId)) {
+    selectedUserIds.value = selectedUserIds.value.filter(id => id !== userId)
+  } else {
+    selectedUserIds.value.push(userId)
+  }
+}
+
+const showPagination = computed(() => {
+  return params.limit === null
```

Comment:
In `useParams` you had `undefined` return type, but here you compare one of the parameters to `null`. `null` != `undefined`. To make this code work for `null`, `undefined` and empty string, you can:
 ```suggestion
   return !params.limit
```